test/consts/zz_generated_gateway.go

```diff
@@ -3,6 +3,7 @@
 package consts
 
 const (
-	GatewayCRDsKustomizeURL = "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0"
-	GatewayRawRepoURL       = "https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v0.5.0"
+	GatewayStandardCRDsKustomizeURL = "github.com/kubernetes-sigs/gateway-api/config/crd/?ref=v0.5.0"
+	GatewayCRDsKustomizeURL         = "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0"
```

Comment:
IMHO this `const` should have experimental in its name to differentiate between this one and the one above it

```suggestion
	GatewayExperimentalCRDsKustomizeURL         = "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0"
```

Reply:
Also considered changing to `Experimental` here, but I did not want to change to much on existing code. Now that gateway API has been split into 2 release channels, I think you are right to explicitly note them.

Reply:
updated. Leave this unresolved because this change affects more than e2e tests.